reporter/sender-awssdk-sqs/src/main/java/zipkin2/reporter/awssdk/sqs/AbstractSender.java

```diff
@@ -63,7 +63,7 @@ abstract class AbstractSender extends Sender {
 
   boolean isAscii(byte[] encodedSpans) {
     for (int i = 0; i < encodedSpans.length; i++) {
-      if (encodedSpans[i] >= 0x80) {
+      if (Byte.toUnsignedInt(encodedSpans[i]) >= 0x80) {
```

Comment:
interesting

Reply:
so byte is signed  [-128 to 127], and 0x80 is exactly -128, the comparison to 0x80 was actually lifting `byte` to `int`

Reply:
thanks, might want to github search the whole org and also openzipkin-contrib in case this bug is elsewhere via copy/pasta!